### ISSUE TITLE
No ldap group after nextcloud snap 17 upgrade

### ISSUE THREAD

**Issue by sbalay-ghtcge** [nextcloud/nextcloud-snap#1268]
Hi,

First of all, sorry for my bad english.

We have a nextcloud server connected with an ldap serveur (with fusion directory frontend), this server was on Nextcloud 16.0.8. This night the server has done an auto update to NextCloud 17.0.3 and all our Ldap group disapeer.

The group list in user tab was empty and my users cant acces files  that are locked by groups.

I have tried to change setttings in ldap configuration but the config seems to be ok, and when i clic on "verify settings and count the group item" the system report me that he had found my groups (143).

**Comment by kyrofa** [comment]
> For now i have rolled back to 16.0.8.

Now that you have done that, you might consider switching to `16/stable` for now, which won't automatically update you to v17 until this has been sorted out.

This sounds like an upstream Nextcloud issue, but when I search upstream for ["ldap" issues I see 659 results](https://github.com/nextcloud/server/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+ldap), so it might be hard to find (although github seems to be having issues at the moment). @kesselb does this ring any bells for you?

Mind if we take a look at your logs? Just paste the output of [this script](https://github.com/nextcloud/nextcloud-snap/wiki/Issue-Debugging-Scripts) into a comment, it's pre-formatted.

**Comment by sbalay-ghtcge** [comment]
Hi,

Thx for the fast reply,

It's a good idea to lock on 16/stable :) (and it's done now), i'm don't know to much how to use snap it's a co worker who left the company who did this install (when i have learnt linux, snap didnt exist yet and i didnt take/have time to learn how to use it).

Here is the logs: [debug_nc_20200226.log](https://github.com/nextcloud/nextcloud-snap/files/4254112/debug_nc_20200226.log)

Best regards

**Comment by kesselb** [comment]
> does this ring any bells for you?

No. Any change to see the nextcloud.log? Probably not bad to change the loglevel to 0 for easier debugging.

**Comment by kyrofa** [comment]
@sbalay-ghtcge can we see the contents of `/var/snap/nextcloud/common/nextcloud/data/nextcloud.log` as well, please? Note that if you've changed your data directory from the default, it will be in there instead. Attach that, don't paste it.

**Comment by sbalay-ghtcge** [comment]
Hi,

Thx for reply, i have a little problem with that file : My users are listed in it and  my file tree too.

I can't give you  a file that contain so much informations.

Best regards

**Comment by kyrofa** [comment]
Not a lot we can do to help you debug this then, I'm afraid. Consider emailing it, perhaps?
